webapp/locales/de.json

```diff
@@ -510,10 +510,10 @@
     "name": "Mein Profil",
     "network": {
       "andMore": "und {number} weitere …",
-      "followedBy": "wird gefolgt von:",
-      "followedByNobody": "wird von niemandem gefolgt.",
-      "following": "folgt:",
-      "followingNobody": "folgt niemandem.",
+      "followedBy": "wird abboniert von:",
+      "followedByNobody": "wird von niemandem abonniert.",
+      "following": "abonniert:",
+      "followingNobody": "abboniert niemandem.",
```

Comment:
> [<img alt="Mogge" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Mogge) **Authored by [Mogge](https://github.com/Mogge)**
_<time datetime="2020-02-26T00:58:28Z" title="Wednesday, February 26th 2020, 1:58:28 am +01:00">Feb 26, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -510,10 +510,10 @@
     "name": "Mein Profil",
     "network": {
       "andMore": "und {number} weitere …",
-      "followedBy": "wird gefolgt von:",
-      "followedByNobody": "wird von niemandem gefolgt.",
-      "following": "folgt:",
-      "followingNobody": "folgt niemandem.",
+      "followedBy": "wird abboniert von:",
+      "followedByNobody": "wird von niemandem abonniert.",
+      "following": "abonniert:",
+      "followingNobody": "abboniert niemandem.",
```


German is a difficult language :) It is `abonniert niemanden`